visidata/addons/tidydata.py

```diff
@@ -6,7 +6,7 @@
 option('melt_value_colname', 'Value', 'column name to use for the melted value')
 
 class MeltedSheet(Sheet):
-    """Perform "melt", the reverse of "pivot", on input sheet."""
+    '''Perform 'melt', the reverse of 'pivot', on input sheet.'''
```

Comment:
Official PEP8 recommendation is to use " as outer quotes in this case, and I agree.